content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -693,7 +675,83 @@ data:
 
 - You can't use ConfigMaps for {{< glossary_tooltip text="static pods" term_id="static-pod" >}}, because the Kubelet does not support this.
 
+### Optional ConfigMaps
+
+A ConfigMap reference may be marked "optional".
+If the ConfigMap is non-existent, the mounted volume will be empty.
```

Comment:
BTW this isn't quite right because there isn't necessarily any mounted volume.

Reply:
Good point, I missed that - have expanded this a bit, and modified the wording in the subsection that is specifically about mounted volumes in https://github.com/kubernetes/website/pull/35160/commits/7e16543b9dbe6771017730199b62e2b99c18803e.